examples/bitbucket/bitbucket-eventlistener-interceptor.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: EventListener
+metadata:
+  name: bitbucket-listener
+spec:
+  serviceAccountName: tekton-triggers-example-sa
+  triggers:
+    - name: bitbucket-triggers
+      interceptors:
+        - bitbucket:
+            secretRef:
+              secretName: githubsecret
+              secretKey: secretToken
+            eventTypes:
+              - repo:refs_changed
+      bindings:
+        - name: bitbucket-binding
```

Comment:
Can we put the binding and the template in this file as opposed being symlinks? (Like how we do in the GitLab example)?